g10_certification_test_kit.gemspec

```diff
@@ -7,9 +7,11 @@ Gem::Specification.new do |spec|
   spec.description   = 'G10 Certification Tests for Inferno'
   spec.homepage      = 'https://github.com/inferno_framework/g10-certification-test-kit'
   spec.license       = 'Apache-2.0'
-  spec.add_runtime_dependency 'inferno_core', '~> 0.1.2'
   spec.add_runtime_dependency 'bloomer', '~> 1.0.0'
   spec.add_runtime_dependency 'colorize', '~> 0.8.1'
+  spec.add_runtime_dependency 'inferno_core', '~> 0.1.2'
+  spec.add_runtime_dependency 'json-jwt'
```

Comment:
It's a good idea to specify versions on these so we don't get bitten by breaking changes. You can generally do `~> whatever_version_is_in_Gemfile.lock`.